Core/Generators/CSharp/CSharpGenerator.cs

```diff
@@ -46,6 +46,13 @@ public override ValueTask<string> Compile(BebopSchema schema, GeneratorConfig co
                 builder.Indent(indentStep).AppendLine();
             }
 
+            if (Config.EmitBinarySchema)
+            {
+                builder.Append("public const byte[] BEBOP_SCHEMA =");
+                builder.AppendLine(Schema.ToBinary().ConvertToString("new byte[] {", "};"));
+                builder.AppendLine();
+            }
+
```

Comment:
This needs to get emitted into the constants class we generate, otherwise it ends up in no mans land which is invalid C#